src/IssueViz.Security.UnitTests/Hotspots/HotspotsList/HotspotsControlViewModelTests.cs

```diff
@@ -41,13 +41,6 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.UnitTests.Hotspots.
     [TestClass]
     public class HotspotsControlViewModelTests
     {
-        [TestInitialize]
-        public void TestInitialize()
-        {
-            // HotspotsControlViewModel needs to be created on the UI thread
-            ThreadHelper.SetCurrentThreadAsUIThread();
```

Comment:
Ideally, none of the tests would require this call as the product code wouldn't be using the VS `ThreadHelper` directly. I've fixed up the VMs so they don't use `ThreadHelper`.